src/bokeh/core/property/factors.py

```diff
@@ -46,9 +45,6 @@
 L2Factor = Tuple(String, String)
 L3Factor = Tuple(String, String, String)
 
-FactorType: TypeAlias = str | tuple[str, str] | tuple[str, str]
-FactorSeqType: TypeAlias = tp.Sequence[str] | tp.Sequence[tuple[str, str]] | tp.Sequence[tuple[str, str]]
```

Comment:
L3 factor type has just been wrong forever AFAICT

Reply:
"Forever" is a strong statement, given that typing in (Python) bokeh is quite new and these types are hardly ever used in practice, because we don't have typings for model properties.